data-prepper-plugins/kinesis-source/src/test/java/org/opensearch/dataprepper/plugins/kinesis/source/KinesisServiceTest.java

```diff
@@ -70,6 +70,7 @@ public class KinesisServiceTest {
     private final String PIPELINE_NAME = "kinesis-pipeline-test";
     private final String streamId = "stream-1";
     private static final String codec_plugin_name = "json";
+    private static final String PIPELINE_IDENTIFIER = "sample-kinesis-pipeline-0123";
```

Comment:
Please generate this as a random string in the `@BeforeEach` setUp method.

Reply:
Thanks @dlvenable . I have made the changes.